timvt/endpoints/tiles.py

```diff
@@ -40,7 +43,7 @@ async def tile(
     epsg = tile_params.tms.crs.to_epsg()
     segSize = (bbox.xmax - bbox.xmin) / 4
 
-    limit = f"LIMIT {MAX_FEATURES_PER_TILE}" if MAX_FEATURES_PER_TILE > -1 else ""
+    limit = f"LIMIT $9" if MAX_FEATURES_PER_TILE > -1 else ""
```

Comment:
what happens when `MAX_FEATURES_PER_TILE` is `-1` ?

Reply:
the real quesiton is: does `q.fetchval` will be happy with an unused variable 9 when `limit=""` ?

Reply:
nope. I guess not.